h/services/__init__.py

```diff
@@ -17,8 +17,7 @@
 from h.services.jwt import JWTService
 from h.services.mention import MentionService
 from h.services.notification import NotificationService
-from h.services.openid_client import OpenIDClientService
-from h.services.orcid_client import ORCIDClientService
+from h.services.oidc import OIDCService
```

Comment:
`OpenIDClientService` and `ORCIDClientService` are gone, replaced with a single new `OIDCService`.